web/src/main/java/org/cbioportal/web/columnstore/StudyViewControllerColumnStore.java

```diff
@@ -0,0 +1,73 @@
+package org.cbioportal.web.columnstore;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.cbioportal.model.AlterationCountByGene;
+import org.cbioportal.model.Sample;
+import org.cbioportal.service.StudyViewService;
+import org.cbioportal.web.config.annotation.InternalApi;
+import org.cbioportal.webparam.StudyViewFilter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestAttribute;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import springfox.documentation.annotations.ApiIgnore;
+
+import javax.validation.Valid;
+import java.util.Collection;
+import java.util.List;
+
+@InternalApi
+@RestController
+@Validated
+@Api(tags = "Study View Column Store")
+public class StudyViewControllerColumnStore {
```

Comment:
maybe name it this way? not a big deal though, we will most likely end up renaming these to something different.

```suggestion
public class StudyViewColumnStoreController {
```